lib/livebook/application.ex

```diff
@@ -203,16 +203,12 @@ defmodule Livebook.Application do
     defp app_specs, do: []
   end
 
-  if Livebook.Config.feature_flag_enabled?(:localhost_hub) do
-    defp insert_development_hub do
-      Livebook.Hubs.save_hub(%Livebook.Hubs.Local{
-        id: "local-host",
-        hub_name: "Localhost",
-        hub_emoji: "🏠"
-      })
-    end
-  else
-    defp insert_development_hub, do: :ok
+  defp insert_personal_hub do
+    Livebook.Hubs.save_hub(%Livebook.Hubs.Personal{
+      id: "personal-hub",
+      hub_name: "My Hub",
+      hub_emoji: "🏠"
+    })
```

Comment:
Isn't this going to override the changes on every startup?

Reply:
Yeahh, that's true. let me change